document/CoreFormat.mdk

```diff
@@ -413,6 +413,44 @@ The `results` array shall be empty if the run that produced the `run` object did
 If the tool was run solely for the purpose of exporting rule metadata (see &sect;[#run-rules]),
 the `results` property shall be absent.
 
+### `toolNotifications` { #run-toolNotifications }
+
+A `run` object may contain a property named `toolNotifications`
+whose value is an array of zero or more `notification` objects (&sect;[#notification]).
+Each element of the array represents a runtime condition detected by the tool.
+The presence within this array of any `notification` object whose `level` property (&sect;[#notification-level]) is `error` shall mean
+that the run failed.
+
+~ IsoNote
+Depending on the nature of the error, a tool that encounters a runtime error might or might not be able to continue running.
+
+If the error occurs in the course of evaluating a rule, the tool might report the error in `toolNotifications`, disable the rule,
+and continue to execute the remaining rules.
+
+If the error occurs outside of the evaluation of a rule, the tool might report
+the error in `toolNotifications` and then halt. The tool might also just exit, either normally or abnormally.
+~
+
+### `configurationNotifications` { #run-configurationNotifications }
+
+A `run` object may contain a property named `configurationNotifications`
+whose value is an array of zero or more `notification` objects (&sect;[#notification]).
+Each element of the array represents a condition relevant to the tool's configuration.
+The presence within this array of any `notification` object whose `level` property (&sect;[#notification-level]) is `error` shall mean
+that the run failed.
+
+~ IsoNote
+Many tools can be parameterized with information about which rules to run, and how they should be configured.
+In some cases, if the configuration information is invalid, the tool can ignore the invalid information
+and continue to run.
+~
+
+~ IsoExample
```

Comment:
Again, can we sneak in the point that this information is targeted towards the person or system responsible for producing tool runs? not the tool owner. not necessarily the tool results consumers (but could be, of course, when a user is running a tool for her own purposes)